cli.go

```diff
@@ -112,7 +112,7 @@ func main() {
 		},
 		cli.StringFlag{
 			Name: "dns",
-			Usage: "Enable the DNS challenge for solving using a provider." +
+			Usage: "Solve a DNS challenge using the specified provider." +
```

Comment:
Yeah that's fine as well. I just wanted to make sure that it clearly makes this point everywhere.